python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2324,6 +2324,35 @@ def _impl(inputs, attr, params, mod):
     return _impl
 
 
+def _unique():
+    def _impl(inputs, attr, params, mod):
+        assert len(inputs) == 1
+        data = inputs[0]
+        [unique, indices, num_uniq] = _op.unique(data, is_sorted=False, return_counts=False)
+        unique_sliced = _op.strided_slice(unique, begin=[0], end=num_uniq, slice_mode="size")
+        return _expr.TupleWrapper(
+            _expr.Tuple([unique_sliced, indices]),
+            2,
+        )
+
+    return _impl
+
+
+def _unique_with_counts():
+    def _impl(inputs, attr, params, mod):
+        assert len(inputs) == 1
+        data = inputs[0]
+        [unique, indices, num_uniq, counts] = _op.unique(data, is_sorted=False, return_counts=True)
+        unique_sliced = _op.strided_slice(unique, begin=[0], end=num_uniq, slice_mode="size")
+        counts_sliced = _op.strided_slice(counts, begin=[0], end=num_uniq, slice_mode="size")
+        return _expr.TupleWrapper(
+            _expr.Tuple([unique_sliced, indices, counts_sliced]),
+            3,
+        )
+
+    return _impl
+
```

Comment:
probably we can share the implementation in two converters, just pass `return_counts` to `_unique()` and _`unique_with_counts()`

Reply:
good suggestion, changed.